sktime/forecasting/base/_meta.py

```diff
@@ -27,6 +30,18 @@ def __init__(self, forecasters, n_jobs=None):
         self.n_jobs = n_jobs
         super(_HeterogenousEnsembleForecaster, self).__init__()
 
+    def _check_regressor(self):
```

Comment:
question: why does this need to be in the base class?
To me, it seems this is a concern that is very specific to stacking.
I wouldn't put it here, all the other descendants get it but don't need it, it seems to me?

Reply:
ok, got it now: there are two descendants that need it, stacking and the new estimator.